### ISSUE TITLE
Can't connect internet after install

### ISSUE THREAD

**Issue by bensmeets** [pivpn/pivpn#80]
Did the install, mostly everything to default. After installation is complete (and reboot), the PI can't connect to the internet anymore. E.g., apt-get update will fail because no server can be found and ping www.google.com doesn't work anymore. (all this worked before installing).

Tried adding dns from google to ethernet and resolve, but nothing unfortunately.

**Comment by bensmeets** [comment]
Had to do with something changing the resolv.conf after pivpn was installed (on each reboot)

**Comment by LifeBasher** [comment]
i have the same problem.... have you been able to resolve it?

**Comment by 0-kaladin** [comment]
can you paste in your install output?  also what distro/version etc?

**Comment by cabsandy** [comment]
I had this problem and the only way to fix it was to change the static domain_name_servers entry in **/etc/dhcpcd.conf** from my router (so 192.168.0.1 in my case) to 8.8.8.8.

Reboot and you should be sorted.

**Comment by AzaIndustries** [comment]
Cabsandy you're a legend!
Been trying to fix this problem for a week, its for part of an assignment so I was super stressed.

**Comment by LifeBasher** [comment]
I had multiple line with the static domain_name_server but event if I changed them all, removed some or what ever I still doesn't have internet when I connect to the VPN :-(

**Comment by brianpadley** [comment]
Had the same problem. Thanks, @cabsandy

**Comment by 0-kaladin** [comment]
@cabsandy @AzaIndustries @brianpadley 
the installer uses your default gateway as the dns server.  this typically works as the gateway is typically a home router who will forward dns requests out of its wan interface to the dns of your internet provide.  In your 3 cases this seems not to be true.  
Can you guys tell me on a normal computer on your local nework, what is the gateway of that computer vs what dns servers does it use?  If they are in fact different then I'd need to ask that. 
Also you'd only get in this state I believe if you used the installer to set a static IP on the raspberry pi.  Can you all confirm this as well?

**Comment by brianpadley** [comment]
I used static IP, I think it's the only option I had.

The DNS server on my pi was the router, and on normal computer it points to OpenDNS.

"Normal" computer:
Default Gateway . . . . . . . . . : 192.168.1.1
DHCP Server . . . . . . . . . . . : 192.168.1.2
DNS Servers . . . . . . . . . . . : 208.67.222.222
                                    208.67.220.220

**Comment by 0-kaladin** [comment]
In this case the pivpn install hosed your dns servers.  the install should not do this any longer so I'll close this.

**Comment by MaybeGoogle** [comment]
I know this is an old issue, but here we go anyway:

I'm having this exact problem (down to the trying to ping google.com first time it failed 😄) but on ubuntu 16.04.3. Now, I have no knowledge of how networking works in a vps, but I'm pretty sure that dhcpcd.conf doesnt do the same thing here... the file is just a sample config.

I've already tried editing resolv.conf; it has some default dns and a google one (8.8.4.4). Removed default, still not working.

Any ideas?
